src/Makefile

```diff
@@ -24,8 +24,8 @@ ifdef NO_PKG_CONFIG
 	ALL_LDFLAGS += -lelf -lz
 else
 	PKG_CONFIG ?= pkg-config
-	ALL_CFLAGS += $(shell $(PKG_CONFIG) --cflags libelf)
-	ALL_LDFLAGS += $(shell $(PKG_CONFIG) --libs libelf)
+	ALL_CFLAGS += $(shell $(PKG_CONFIG) --cflags libelf --cflags zlib)
```

Comment:
would just `--cflags libelf zlib` work? no need to repeat --cflags flag?

Reply:
Right.  I tested it (with 0.1.0) and also `man pkg-config` suggests so.  I agree it will be better; updated.